mlflow/utils/annotations.py

```diff
@@ -34,8 +55,10 @@ def f(api: C) -> C:
 
 
 def _experimental(api: C, api_type: str) -> C:
+    indent = _get_min_indent_of_docstring(api.__doc__)
     notice = (
-        f"    .. Note:: Experimental: This {api_type} may change or "
+        indent
+        + f".. Note:: Experimental: This {api_type} may change or "
         + "be removed in a future release without warning.\n\n"
```

Comment:
```suggestion
        f".. Note:: Experimental: This {api_type} may change or "
        "be removed in a future release without warning.\n\n"
```

Reply:
Looks like `()` syntax for multiline strings needs the first arg to be a string, not a variable. I opted for `+` over making the `indent` variable an fstring.